src/new-client/src/sw.js

```diff
@@ -0,0 +1,77 @@
+/// <reference lib="webworker" />
+/* eslint-disable no-restricted-globals */
+
+// This service worker can be customized!
+// See https://developers.google.com/web/tools/workbox/modules
+// for the list of available Workbox modules, or add any other
+// code you'd like.
+// You can also remove this file if you'd prefer not to use a
+// service worker, and the Workbox build step will be skipped.
+
+import { clientsClaim } from "workbox-core"
+import { ExpirationPlugin } from "workbox-expiration"
+import { precacheAndRoute, createHandlerBoundToURL } from "workbox-precaching"
+import { registerRoute } from "workbox-routing"
+import { StaleWhileRevalidate } from "workbox-strategies"
+
+clientsClaim()
```

Comment:
What's tldr; on what this does?

Reply:
Hey stranger :)

Reply:
![Magic!](https://media.giphy.com/media/ujUdrdpX7Ok5W/giphy.gif)

Reply:
> ![Magic!](https://camo.githubusercontent.com/4cd06dd983ef39c6d8f43c0be1a5eadd4078a3dbf5d33bb336aecf91fd7b22bd/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f756a556472647058374f6b35572f67697068792e676966)
> 
> Just kidding, it's just that workbox kinda feels like magic 🙂, here you go: https://developers.google.com/web/tools/workbox/modules/workbox-core#clients_claim

🧙